src/main/java/com/imsweb/mph/HematoDbDTO.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2016 Information Management Services, Inc.
+ */
+package com.imsweb.mph;
+
+public class HematoDbDTO {
```

Comment:
Do you expect people to use this class, or is it just used internally? If it's just internally, it might be better to move it to a "mph.internal" package to make it clear it's not intended to be exposed...